vuln/npm/472.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": 472,
+  "title": "SQL Injection",
```

Comment:
This seems a little off, given that no SQL database is involved here. Otherwise LGTM.

Reply:
MongoDB injection?

Reply:
Sounds better but I am not sure it maps well to CWE field in H1.